package.json

```diff
@@ -8,12 +8,11 @@
   "typings": "es5/index.d.ts",
   "sideEffects": false,
   "scripts": {
-    "test:js": "jest",
+    "test:js": "jest && codecov",
```

Comment:
`test:js` shouldn't run coverage. It slows down test running and makes stack traces unintelligible, which is not what I want for the script that people will be running to validate their changes during local development. Also during local dev there's not much point in uploading the results using the `codecov` command. In fact I'd say only the `verify` script aught to do that, since that's the script I expect CI to run.